.semaphore/semaphore.yml

```diff
@@ -17,7 +17,18 @@ blocks:
           - npx lerna bootstrap
           - cache store node-modules-$SEMAPHORE_GIT_BRANCH node_modules
           - cache store cypress-$SEMAPHORE_GIT_BRANCH ~/.cache/Cypress
-
+  - name: "🔗 Storybook link"
+    task:
+      secrets:
+        - name: paprika-env
+      prologue:
+        commands:
+          - checkout
+          - cache restore node-modules-$SEMAPHORE_GIT_BRANCH
+      jobs:
+        - name: Insert Storybook link into PR Body Message
```

Comment:
So many capitals 🙈, if you need to address something else, can change this to
Insert Storybook link into PR body message :)

Reply:
Otherwise its okay, if you have nothing to address